services/src/test/scala/cromwell/services/metadata/impl/ArchiverSpec.scala

```diff
@@ -0,0 +1,29 @@
+package cromwell.services.metadata.impl
+
+import cromwell.services.metadata.WorkflowQueryKey.EndDate
+import cromwell.services.metadata.impl.archiver.{ArchiveMetadataSchedulerActor, DeleteMetadataActor}
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.must.Matchers
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import scala.language.postfixOps
+import java.time.OffsetDateTime
+
+class ArchiverSpec extends AnyFlatSpec with Matchers {
```

Comment:
This test is not really necessary but I have trust issues with time computations

Reply:
Given that you are now hopefully convinced, is it still worth having these tests. It seems to me like they are still validating a small % of custom Cromwell code even alongside the "did java implement it right?" paranoia?